recipes/coinbonmin/build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
```

Comment:
This shebang is not necessary, ``conda-build`` already handles it, could you please remove it?
```suggestion
```

Reply:
ok, done